README.md

```diff
@@ -17,7 +17,7 @@ Bundler helps manage dependencies for Ruby projects. Find more info here: <http:
 Add this gem to your `Gemfile`:
 
 ```ruby
-gem "opentok", "~> 2.4"
+gem "opentok", "~> 2.4.1"
```

Comment:
I would recommend going with `>= 2.4.0` based on this docs: http://bundler.io/v1.5/gemfile.html. If we think some previous versions are reliable, we could do a lower minimum version.